src/Particle.h

```diff
@@ -14,12 +14,12 @@ class Particle {
 
 private:
   /**
-   * Position of the particle
+   * Position of the particle.[R_earthorbit]
```

Comment:
You will need to change these units with assignment 2, but it is an excellent documentation idea to document the units (speaking here from experience 🤓😇)